spec/cellular/backends/cool_sms_spec.rb

```diff
@@ -14,6 +14,19 @@
     }
   }
 
+  let(:payload){
```

Comment:
having trouble adding spaces after, sense I am using a trailing spaces plugin. That removes all trailing spaces when saving a file :) hope that is ok